test/asm/builtin-overwrite.err

```diff
@@ -1,57 +1,57 @@
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(7):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(8):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(11):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(12):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(16):
-    Built-in symbol '__LINE__' cannot be purged
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(17):
-    Built-in symbol '__LINE__' cannot be purged
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(20):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(21):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(24):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(25):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(28):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(29):
-    '__LINE__' already defined at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(32):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(33):
-    '__LINE__' already defined as constant at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(36):
-    '__LINE__' already defined as non-EQUS at <builtin>
-error: builtin-overwrite.asm(42) -> builtin-overwrite.asm::tickle(37):
-    '__LINE__' already defined as non-EQUS at <builtin>
+error: builtin-overwrite.asm(43) -> builtin-overwrite.asm::tickle(7):
+    '__UTC_YEAR__' already defined as constant at <command-line>
```

Comment:
Wait, why is it not reported as `<builtin>`?

Reply:
Because its value comes from the environment variable `SOURCE_DATE_EPOCH`.

Reply:
I guess this has been here for a while, but I found it surprising, considering `<command-line>` normally means that it was defined using `-D`. It makes sense in a way, but this might be against PoLA. Idk.

Reply:
It's not new to this PR, anyway, and applies to the other time symbols too.